docs/gallery_tests/test_plot_wind_barbs.py

```diff
@@ -0,0 +1,30 @@
+# Copyright Iris contributors
+#
+# This file is part of Iris and is released under the LGPL license.
+# See COPYING and COPYING.LESSER in the root of the repository for full
+# licensing details.
+
+# Import Iris tests first so that some things can be initialised before
+# importing anything else.
+import iris.tests as tests
```

Comment:
```suggestion
# Import Iris tests first so that some things can be initialised before
# importing anything else.
import iris.tests as tests  # isort:skip
```

You're right that none of the other gallery tests have this yet, but that's an oversight and we've just got lucky so far. The statement about importing `iris.tests` first will stop being true the moment we add another import that isort thinks should go before it.